packages/data/src/world/mm/stone_tower_temple.yml

```diff
@@ -83,21 +83,21 @@
 "Stone Tower Temple Water Room":
   dungeon: ST
   exits:
-    "Stone Tower Temple Center": "has_mask_zora || (short_hook_anywhere && soul_enemy(SOUL_ENEMY_DEXIHAND))"
+    "Stone Tower Temple Center": "has_mask_zora || (short_hook_anywhere && soul_enemy(SOUL_ENEMY_DEXIHAND)) || (has_tunic_zora && has_iron_boots && can_hookshot)"
```

Comment:
the iron boots should also account for the dexihand soul, and is short hook not enough to get the chest? if not, irons + short hook anywhere would also work.

Reply:
You don't need the dexihand. You hookshot to the chest from underwater, and short hook is not long enough. You're right about irons + short hook anywhere though.

Reply:
Oh i understand.